test/e2e/upload.test.js

```diff
@@ -1,15 +1,200 @@
-
 const { assert, driver } = require('vl-ui-core').Test.Setup;
 const VlUploadPage = require('./pages/vl-upload.page');
+const path = require('path');
+const Express = require("express");
+const Multer = require("multer");
 
 describe('vl-upload', async () => {
     const vlUploadPage = new VlUploadPage(driver);
+    let fileUploadServer;
+    const uploadServerPort = 8888;
+    const uploadServerPath = "/post";
+    
+    beforeEach(async () => {
+    	fileUploadServer.reset();
+    	await vlUploadPage.clearAllUploads();
+    });
+    
+    before((done) => {
+    	vlUploadPage.load().then(() => {
+    		vlUploadPage.changeAllUploadUrlsTo(`http://localhost:${uploadServerPort}${uploadServerPath}`).then(() => {
+    	    	fileUploadServer = new FileUploadServer(uploadServerPort, uploadServerPath);
+    	    	fileUploadServer.start(done);
+    		});
+    	});
+    });
```

Comment:
```suggestion
    before((done) => {
    	vlUploadPage.load().then(() => {
    		vlUploadPage.changeAllUploadUrlsTo(`http://localhost:${uploadServerPort}${uploadServerPath}`).then(() => {
    	    	fileUploadServer = new FileUploadServer(uploadServerPort, uploadServerPath);
    	    	fileUploadServer.start(done);
    		});
    	});
    });
    
    beforeEach(async () => {
    	fileUploadServer.reset();
    	await vlUploadPage.clearAllUploads();
    });
```
Misschien iets leesbaarder als de before eerst komt?

Reply:
ok